sympy/stats/crv_types.py

```diff
@@ -341,8 +350,37 @@ def pdf(self, x):
         alpha, beta = self.alpha, self.beta
         return x**(alpha - 1) * (1 - x)**(beta - 1) / beta_fn(alpha, beta)
 
-    def sample(self):
-        return random.betavariate(self.alpha, self.beta)
+    def _sample_python(self, size):
+        if size == ():
```

Comment:
`if not size`? sounds more intuitive to me.

Reply:
Yes, thanks!

Reply:
`if not size` is able to capture cases such as `size=[]`.

Also consider `if len(size) == 0:`, despite being longer and less pythonic, it has the advantage of being more readable (readers understand immediately that it is a tuple or array).